south_migrations/0001_initial.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
```

Comment:
This might be beyond anal on my part, but might it make sense to explicitly track this as a re-name, and the new migrations/0001_initial.py as a strictly new file by putting it in a separate commit? 

(Feel free to ignore this if you don't want to amend the commit, this is mostly my inner OCD talking).